target/linux/ar71xx/image/generic.mk

```diff
@@ -196,6 +209,45 @@ define Device/mr16
 endef
 TARGET_DEVICES += mr16
 
+define Device/mr1750
+  DEVICE_TITLE := OpenMesh MR1750
+  DEVICE_PACKAGES := om-watchdog kmod-ath10k ath10k-firmware-qca988x
+  BOARDNAME = MR1750
+  FILESYSTEMS := squashfs
+  BLOCKSIZE := 4 8 16 64 128 256
+  KERNEL := kernel-bin | lzma | uImage lzma
+  IMAGES := sysupgrade.bin factory.bin
+  IMAGE/sysupgrade.bin/squashfs := append-rootfs | pad-rootfs | combined-ext-image board=MR1750 rootfs=$$$$@
```

Comment:
> CONFIG_TARGET_EXT4_BLOCKSIZE=4096
CONFIG_TARGET_EXT4_BLOCKSIZE_4K=y
CONFIG_TARGET_EXT4_RESERVED_PCT=0
CONFIG_TARGET_IMAGES_GZIP=y
CONFIG_TARGET_ROOTFS_EXT4FS=y
CONFIG_TARGET_ROOTFS_PARTSIZE=256

Correct me if I'm wrong, but this is neither is the default config nor the config used for the prebuild LEDE images.

If you have to change the rootfs filesystem type, either something is wrong with the openmesh support in LEDE or you are having some really special kind of use case.

Most likely I miss the point here.